### PR TITLE
Added GMK Hazakura colorway

### PR DESCRIPTION
**Pull request by HorrorTroll**
## Description

Added GMK Hazakura colorway, I was asking for permission and he approved for me to open pr.

image

### CODE REVIEW

File: src/scss/colorways.scss [hunk 1]
```diff
@@ -717,6 +717,31 @@
   // placeholder
 }
 
+.gmk-hazakura-key {
+  background: $color-gmk-abs-WS1;
+  color: $color-pantone-2346C;
```
Comment (noroadsleft) [inline src/scss/colorways.scss:722]:
```suggestion
  color: darken($color-pantone-2346C, 10%);
```

File: src/scss/colorways.scss [hunk 2]
```diff
@@ -717,6 +717,31 @@
...
+  input,
+  .key-contents {
+    background: darken($color-gmk-abs-WS1, 7%);
+    color: lighten($color-pantone-2346C, 0%);
+    border-color: mix($color-gmk-abs-WS1, lighten($color-pantone-2346C, 0%));
+  }
+}
+.gmk-hazakura-mod {
+  background: $color-pantone-182C;
+  color: $color-pantone-2346C;
```
Comment (noroadsleft) [inline src/scss/colorways.scss:732]:
```suggestion
  color: darken($color-pantone-2346C, 10%);
```

File: src/scss/colorways.scss [hunk 3]
```diff
@@ -717,6 +717,31 @@
...
+  input,
+  .key-contents {
+    background: darken($color-pantone-182C, 5%);
+    color: lighten($color-pantone-2346C, 0%);
+    border-color: mix($color-pantone-182C, lighten($color-pantone-2346C, 0%));
+  }
+}
+.gmk-hazakura-accent {
+  background: $color-pantone-2346C;
+  color: $color-pantone-182C;
```
Comment (noroadsleft) [inline src/scss/colorways.scss:742]:
```suggestion
  color: lighten($color-pantone-182C, 10%);
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
I was dissatisfied with the color contrast as originally submitted. It looks great but I found it hard to read, and I have 20/20 vision.

before / after:
image

(I also may iterate on this in another PR.)

**Comment by HorrorTroll** [comment]
Thanks, I'm just resolving those issue. Could you please re-check?

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
That'll do, thanks.

**Pull request merged**